packages/harpguru-core/src/components/option/option.tsx

```diff
@@ -57,6 +70,22 @@ export const Option = (props: OptionProps): React.ReactElement => {
     outputRange: isUpdated ? [2, 1] : [1, 1],
   })
 
+  const activeIdPos = orderedOptionIds.indexOf(activeOptionId)
+  const { length: listLength } = orderedOptionIds
+
+  const extendedList = [
+    ...orderedOptionIds,
+    ...orderedOptionIds,
+    ...orderedOptionIds,
+  ]
+  const innerActiveIdPos = activeIdPos + listLength
+
+  const { [innerActiveIdPos + 2]: inactiveOptionId1 } = extendedList
```

Comment:
These names are not very meaningful. I worry that more meaningful names would be just as hard to interpret.

This is going to become a moot point when the option values sections are reimplemented to be a rolladex format, but it's worth recording in the short term.

Reply:
Raised as tech debt in #20